plugin/completion.py

```diff
@@ -40,11 +41,100 @@
 }
 
 
+class LspAutoCompleteListener(sublime_plugin.ViewEventListener):
```

Comment:
This listener has no state. Just put the callbacks in `CompletionHandler`. Generally I would like to move towards a single ViewEventListener for the whole plugin.

Reply:
Moved to CompletionHandler